core/benches/shredder.rs

```diff
@@ -9,13 +9,16 @@ use {
     solana_entry::entry::{create_ticks, Entry},
     solana_ledger::shred::{
         max_entries_per_n_shred, max_ticks_per_n_shreds, ProcessShredsStats, Shred, ShredFlags,
-        Shredder, MAX_DATA_SHREDS_PER_FEC_BLOCK, SIZE_OF_DATA_SHRED_PAYLOAD,
+        Shredder, MAX_DATA_SHREDS_PER_FEC_BLOCK,
     },
     solana_perf::test_tx,
     solana_sdk::{hash::Hash, packet::PACKET_DATA_SIZE, signature::Keypair},
     test::Bencher,
 };
 
+// Equivalent to ledger::shred::legacy::ShredData::CAPACITY.
+const LEGACY_SHERD_DATA_CAPACITY: usize = 1051;
```

Comment:
> s/SHERD/SHRED/

Fixed, thanks.

> Can we continue to expose this value from solana_ledger::shred prefixed with something like FOR_TESTING_ rather than referencing the original location in a comment?

Similar issues as in the other comment: https://github.com/solana-labs/solana/pull/25237#discussion_r884337032